packages/react-select/src/stories/SelectControlled.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Select } from '../index';
+import { useId } from '@fluentui/react-utilities';
+
+export const Controlled = () => {
```

Comment:
I think we also need an example with `onChange` since I guess the most common use case of a select is to know when someone has made a choice

Reply:
The reason I didn't add it in this PR was b/c I was somewhat hoping to wait until the onChange RFC was merged. I do plan to include that at some point, though. I could update the Select epic with a checklist that includes updating the controlled example with the standardized `onChange` handler -- does that sound like a good approach?

Reply:
yes either that or a separate issue, it's quite important for this component

Reply:
@ling1726 here's the issue: https://github.com/microsoft/fluentui/issues/21149